CODEOWNERS

```diff
@@ -223,15 +218,14 @@
 # The following filepaths should be sorted so that more specific paths occur
 # after the less specific paths, otherwise the ownership for the specific paths
 # is not properly picked up in Github.
-* @cilium/tophat
-/AUTHORS @cilium/tophat
+/AUTHORS @cilium/contributing
```

Comment:
Yeah given the contribution history to files like this I think it makes sense to make it owned by another group responsible for releases.